tests/Umbraco.Tests.Integration/Umbraco.Core/Services/DocumentNavigationServiceTests.Rebuild.cs

```diff
@@ -10,7 +10,12 @@ namespace Umbraco.Cms.Tests.Integration.Umbraco.Core.Services;
 public partial class DocumentNavigationServiceTests
 {
     [Test]
-    public async Task Structure_Can_Rebuild()
+    public async Task Structure_Can_Rebuild() => await Perform_Structure_Can_Rebuild();
+
+    [Test]
+    public async Task Structure_Can_Rebuild_Multiple_Times() => await Perform_Structure_Can_Rebuild(2);
+
+    private async Task Perform_Structure_Can_Rebuild(int numberOfRebuilds = 1)
```

Comment:
Why not use a TestCase attribute? You can even specify the name of the test inside the testrunner

```csharp
[TestCase(1, TestName="Structure_Can_Rebuild")]
[TestCase(2, TestName="Structure_Can_Rebuild_MultipleTimes")]
public async Task Structure_Can_Rebuild(int numberOfRebuilds)
```

Reply:
Good idea, I've applied that.